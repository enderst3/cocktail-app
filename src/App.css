.Header {
  text-align: center;
}

.SearchBar {
  text-align: center;
}

.Results {
  text-align: center;
  cursor: pointer;
}

.Results:hover {
  background-color: #eee;
}

.RecipeDetail {
  text-align: center;
}

.DrinkName {
  text-align: center;
}

.CocktailImage {
  text-align: center;
}

p {
  font-size: 16px;
}

body {
	background-image: url('/cocktail2.jpg');
	background-position: center;
	background-repeat: no-repeat;
  background-size: cover;
  background-attachment: fixed;
}

.App {
  opacity: 0.8;
  padding: 50px;
}
