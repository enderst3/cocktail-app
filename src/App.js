import React, { Component } from 'react';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <div id='header'>
        <h1>Cocktail Search App</h1>
        </div>
        <div id='searchBar'>
          <h3>Enter a Drink Name:</h3>
          <input id='drink_input' placeholder='Enter Drink...'/>
          <button>Search</button>
        </div>
        <div id='results'></div>
      </div>
    );
  }
}

export default App;
