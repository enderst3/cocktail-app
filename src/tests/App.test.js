import React from 'react'
import App from '../App'
import { shallow } from 'enzyme'

/* global it describe expect beforeEach */

describe('SearchBar', () => {
  let wrapper, app

  beforeEach(() => {
    wrapper = shallow(<App />)
    app = wrapper.instance()
  })

  it('properly performs onSearchTermInput', () => {
    app.onSearchTermInput({target: {value: 'gin'}})
    expect(wrapper.state().searchTerm).toBe('gin')
  })
})
