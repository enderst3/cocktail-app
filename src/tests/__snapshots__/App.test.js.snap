// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App checks a snapshot 1`] = `
<div
  className="App"
>
  <div
    className="col-md-8 col-md-offset-2"
  >
    <div
      className="panel panel-default"
      id={undefined}
    >
      <div
        className="panel-body"
      >
        <div
          className="Header jumbotron"
        >
          <h1>
            Cocktail Search App
          </h1>
        </div>
        <div
          className="SearchBar panel panel-default"
          id={undefined}
        >
          <div
            className="panel-body"
          >
            <h3>
              Enter a Drink Name:
            </h3>
            <form>
              <input
                className="form-control"
                id="drink-input"
                onChange={[Function]}
                onKeyPress={[Function]}
                placeholder="Enter Drink Name..."
                type="text"
                value=""
              />
            </form>
          </div>
        </div>
        <div
          className="StartDiv"
        />
        <ul
          className="DrinkList list-group"
        />
      </div>
      <div
        className="panel-footer"
      >
        Â©2017 My Cocktails Inc.
      </div>
    </div>
  </div>
</div>
`;
